test/EmployeeVestingToken.test.ts

```
@@ -0,0 +1,351 @@
+            .releaseToBatch([beneficiary1.address, beneficiary2.address]);
+
+        expect(await amdToken.balanceOf(beneficiary1.address)).to.equal(
+            scheduleAmount1
+        );
+        expect(await amdToken.balanceOf(beneficiary2.address)).to.equal(
+            scheduleAmount2
+        );
+    });
+});
+
+import { ethers } from "hardhat";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ethers'. Consider to delete it or rename it.

'ethers' is defined here:


(lint/suspicious/noRedeclare)